src/chapters/sql-part2/exercises.rst

```diff
@@ -0,0 +1,107 @@
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Our third table is ``garden_bed``. ``garden_bed`` needs to have four columns:
+
+#. ``space_number``, which will increment as we plant new plants.
+#. ``date_planted`` of type ``DATE`` for the day that we planted the plant on.
+#. ``doing_well`` of type ``BOOL`` and tells us if the plant is healthy and growing.
+#. ``plant_id`` is the foreign key that connects ``garden_bed`` to ``plant``.
+
+In a new query tab, use ``CREATE TABLE`` to make the ``garden_bed`` table.
+The data for the ``garden_bed`` table is in the ``garden.csv`` file. You can download it directly from Github.
+
+Try out some joins!
+-------------------
+
+Before we jump in with our subqueries and more complex queries, let's practice our joins from the previous chapter.
+After you write each query, look at the result set and reflect on why that information might be helpful.
+If you needed this database to manage your own garden, what would you use the result set for?
+
+Inner Join
+^^^^^^^^^^
+
+Use an inner join on ``seeds`` and ``garden_bed`` to see which plants are in our garden bed that we also have seeds for.
+
+Left Join
+^^^^^^^^^
+
+Write a query that joins ``seeds`` and ``garden_bed`` with a left join to see all of the seeds we have and any matching plants in the garden bed.
+
+Right Join
+^^^^^^^^^^
+
+Write a query that joins ``seeds`` and ``garden_bed`` with a right join to see all the plants in the garden bed and any matching seeds we have.
+
+Full Join
+^^^^^^^^^
+
+Write a query that joins ``seeds`` and ``garden_bed`` with a full join.
+
+.. admonition:: Note
+
+   In the previous chapter, we learned that MySQL does not support full joins.
+   Review the section on :ref:`full joins <fullouterjoin>` to see how this could be done!
+
+Sub-Queries and Complex Queries
+-------------------------------
+
+#. When we were writing our joins, you may have noticed that the information that was most helpful to you (the ``plant_name``) was missing from the result set! Write a query that gets the name of the plant by joining the ``plant`` table on the result set of the inner join query above. *Hint: Open the query tab with the inner join query and copy it into a new query tab to start. Once you have your inner join setup in a new query tab, it will be easier to write your subquery.*
+#. Let's say our ``plant`` table is so large that we have no idea what the ``plant_id`` for a hosta is. All we know is that there is definitely a row for hostas in the ``plant`` table. Write a query that will insert a new row into our ``seeds`` table. This new row needs to show that we received 100 hosta seeds that will expire on 08/05/2020 (so no need to reorder!) and for the ``plant_id``, use a query *inside* the INSERT statement to get the appropriate ID for hostas. *Hint: In order to get the 'plant_id' of a hosta, you can use the following query in one line 'SELECT plant_id FROM plant WHERE (plant_name LIKE 'Hosta')' inside the 'VALUES' of the 'INSERT' statement.*
```

Comment:
I think we can we add *``syntax highlighting`` inside italics*.

Reply:
The escape wasn't working too well so I changed the italics formatting to just highlighting "Hint"